core/tests/test_utils.py

```diff
@@ -865,6 +868,39 @@ def post_json(
         self.assertEqual(json_response.status_int, expected_status_int)
         return self._parse_json_response(json_response, expect_errors)
 
+    def post_blob(self, url, payload, expected_status_int=200):
+        """Post a BLOB object to the server; return the received object.
+
+        Args:
+            url: str. The URL to which BLOB object in payload should be sent
+                through a post request.
+            payload: bytes. Binary data which needs to be sent.
+            expected_status_int: int. The status expected as a response of post
+                request.
+
+        Returns:
+            dict. Parsed JSON response received upon invoking the post request.
+        """
+        data = payload
```

Comment:
This is not supposed to be a generic method, no handler in Oppia takes blobs for input except for Classifier Handlers. There we have sign/auth based authentication mechanism for security. There is no general mechanism because it isn't required. Perhaps, this should go in ClassifierTestBase since we have a separate base class now.

Reply:
Yup, putting it in the more restricted test base sgtm. Maybe also add some notes on which handlers (or types of handlers) it should be used for.

Reply:
Moved to ClassifierTestBase

Reply:
Added comment.